src/service/poc_mobile.proto

```diff
@@ -156,11 +156,14 @@ message data_transfer_session_req_v1 {
   uint64 upload_bytes = 2;
   uint64 download_bytes = 3;
   data_transfer_radio_access_technology radio_access_technology = 4;
+  string session_id = 5;
+  bytes oui = 6;
```

Comment:
this is a different OUI than the integer OUI identifier data transfer users of the IoT network have assigned to them i take it?